src/index.js

```diff
@@ -98,4 +101,17 @@ app.get('/deletePledge/:pledgeId', async ({ params: { pledgeId } }, res) => {
   }
 });
 
+async function findNewNotifications() {
+  // notify for pledges that have expired
+  const expiredPledges = await db.findAllPledgesExpiredToNotify();
+  expiredPledges.map(async p => {
+    await postOnSlackMultipleChannels({
+      text: `pledge ${p.content} expired just now`
+    }, [p.requester, p.performer]);
+    await db.setExpiredNotificationAsSentOnPledge(p.id);
+  });
+}
+
+setInterval(findNewNotifications, 5 * 1000);
```

Comment:
hey, this isn't a minute! it's only 5 seconds....
This may be an issue as you're not waiting for `findNewNotifications` to actually finish: `expiredPledges.map(...` returns an array of promises which is not `await`ed by anyone.

If it takes longer than 5 seconds to complete `findNewNotifications` you'll eventually send multiple notifications (or may not, I don't know how the `db` works)